### DISCUSSION TITLE
Restarting config file only if no error are detected?

### DISCUSSION THREAD

**Discussion by mahaaaham** [i3/i3#5850] [Q&A]
Hi!

Is there a way to allow the restart of a config i3 file only if no error is detected? Sometimes, my restart makes it unusable, and I don't know how to know before the restart what are my mistakes.

Thank you for your help!

**Comment by orestisfl** [comment]
Try `i3 -C && i3-msg reload` or bind it to a key:
```
bindsym $mod+Shift+c exec --no-startup-id "i3 -C && i3-msg reload"
```
